build.rs

```diff
@@ -0,0 +1,56 @@
+use rerun_except::rerun_except;
+
+use std::env;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+static BOEHM_REPO: &str = "git://github.com/ivmai/bdwgc.git";
+static BOEHM_ATOMICS_REPO: &str = "git://github.com/ivmai/libatomic_ops.git";
+static BOEHM_DIR: &str = "bdwgc";
+static POINTER_MASK: &str = "-DPOINTER_MASK=0xFFFFFFFFFFFFFFF8";
+
+fn run<F>(name: &str, mut configure: F)
+where
+    F: FnMut(&mut Command) -> &mut Command,
+{
+    let mut command = Command::new(name);
+    let configured = configure(&mut command);
+    println!("Executing {:?}", configured);
```

Comment:
As Edd wil attest, I'm allergic to this sort of `println`: it tends to add noise for no obvious gain. I can be convinced otherwise, but it has to be done each time I'm afraid!

Reply:
He's also allergic to `dbg!` ;)

Reply:
Okay, I've removed it: 715dff773ca954ad830a2561926b45d66226b857

Reply:
Allergy resolved :)